src/main/webapp/app/shared/metis/post/post.component.html

```diff
@@ -25,9 +25,8 @@
                     [routerLink]="contextInformation.routerLinkComponents"
                     (click)="onNavigateToContext($event)"
                     routerLinkActive="active"
+                    >{{ contextInformation.displayName + ':' }}</a
                 >
```

Comment:
```suggestion
                    >
                        {{ contextInformation.displayName + ':' }}
                    </a>
```
This is the code style that was also used below in this file (lines 40 -42). We should be consistent here, the suggested one looks better imo.

Edit: Just noticed that this is exectly how the file looked before. What is the reason behind this change?

Reply:
The difference is if the whitespace between `:` and the title is included in the link.
Current solution:

<img width="317" alt="Bildschirmfoto 2021-11-23 um 16 09 23" src="https://user-images.githubusercontent.com/41366522/143050151-5af7bba6-9fca-4990-a61b-0627f50995d9.png">

Suggested solution:
<img width="325" alt="Bildschirmfoto 2021-11-23 um 16 09 59" src="https://user-images.githubusercontent.com/41366522/143050108-c4d8b425-a090-45b6-9510-eae4dfdf6a5f.png">

I want the current version, but I will move the entire tag into line 28. It was not intended that `>` is in a new line.

Reply:
This is btw exactly the case where HTML renders one or multiple newlines as one space character.

You could move the colon out of the Angular brackets btw. Not sure if you agree with that style change, please see how it looks codewise.

Reply:
@julian-christl thanks for the explanation, do you agree with the changes in d89cc69?

Reply:
Yes, that's what I meant with the colon.

inlining the end tags is not possible with Angular, is it? (e.g. line 39 and 40)

Reply:
nope, I tried it several times, the linter always changed it. I just recognized that, after I got very confused :D

Reply:
Sounds good to me. I think the linter doesn't like it because the starting tag is not inline.